### ISSUE TITLE
Software list

### ISSUE THREAD

**Issue by OloOcki** [OloOcki/awesome-citygml#11]
Great initiative from the community to extend the repo by the compatible software list. 

What do you think? Shall we tag them with additional info following the idea of the dataset list?

Say:

Tags:
- Free: :free:
- Somewhat free (e.g., for academic purposes): :blue_book:
- Proprietary:  :moneybag:

Viewers:
- 3DCityDB-Web-Map-Client Cesium-based 3D viewer and JavaScript API for the 3D City Database
[:free:]
[Code] [Docker]

**Comment by thomashkolbe** [comment]
Good idea. But please differentiate between ‘free‘ software and Open Source software. In the latter case, it would even be good, if the respective license would be named (e.g. GPG V3 or Apache etc.).

**Comment by thomashkolbe** [comment]
I suggest to also include the supported CityGML versions and encodings (e.g. CityGML 2.0/GML or CityGML 2.0/CityJSON 1.0). If a software supports multiple CityGML versions and encodings all supported combinations should be enumerated explicitly.

**Comment by b-schwab** [comment]
Yes, tags would be helpful to the reader for a faster overview.
Concerning the open source licenses, I would suggest using the [SPDX identifiers](https://spdx.org/licenses/) (see the [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) list as an example).

Moreover, a brief explanation of the difference between a conceptual model and an encoding in the first paragraph might help beginners to understand this relevant detail.

**Comment by OloOcki** [comment]
Thank you for your insightful remarks. We should definitely include them while adding an additional information.

**Comment by OloOcki** [comment]
Please find the initial version of the lists with tags:
https://github.com/OloOcki/awesome-citygml/blob/main/software.md

Still, many tags to add, and the brief introduction concerning versions and encodings, mentioned by @b-schwab, needs to be written.

Please do not hesitate to directly refine the markdown text or write remarks in this comment section.
